containers/pom.xml

```diff
@@ -91,19 +94,37 @@
         </property>
       </activation>
       <modules>
+        <module>cassandra-seed-provider</module>
+        <module>cassandra-docker</module>
+        <module>container-tests</module>
+        <module>hawkular-deployer-docker</module>
+        <module>hawkular-metrics-openshift-integration</module>
+        <module>metrics-api-jaxrs-1.1-openshift</module>
         <module>hawkular-metrics-docker</module>
       </modules>
     </profile>
     <profile>
       <id>eap-containers</id>
       <activation>
-        <property>
-          <name>!wildfly-containers</name>
-        </property>
+        <activeByDefault>true</activeByDefault>
```

Comment:
This should be default behavior, activate the `containers` submodule with the EAP module to be built. When `-Pwildfly-containers` is activated then it will build the Wildfly variant.